test/functional/core/job_spec.lua

```diff
@@ -179,28 +189,47 @@ describe('jobs', function()
   end)
 
   it("will not buffer data if it doesn't end in newlines", function()
-    if helpers.pending_win32(pending) then return end  -- TODO: Need `cat`.
     if os.getenv("TRAVIS") and os.getenv("CC") == "gcc-4.9"
       and helpers.os_name() == "osx" then
       -- XXX: Hangs Travis macOS since e9061117a5b8f195c3f26a5cb94e18ddd7752d86.
       pending("[Hangs on Travis macOS. #5002]", function() end)
       return
     end
 
-    nvim('command', "let j = jobstart(['cat', '-'], g:job_opts)")
+    nvim('command', "let j = jobstart(has('win32') ? ['find', '/v', ''] : ['cat', '-'], g:job_opts)")
     nvim('command', 'call jobsend(j, "abc\\nxyz")')
-    eq({'notification', 'stdout', {0, {'abc', 'xyz'}}}, next_msg())
+    eq({'notification', 'stdout', {0, iswin() and {'abc', ''} or {'abc', 'xyz'}}}, next_msg())
     nvim('command', "call jobstop(j)")
+    if iswin() then
+      eq({'notification', 'stdout', {0, {'xyz', ''}}}, next_msg())
+    end
     eq({'notification', 'stdout', {0, {''}}}, next_msg())
+    -- TODO jobstop() exit code is unreliable on Windows
+    if helpers.pending_win32(pending) then return end
     eq({'notification', 'exit', {0, 0}}, next_msg())
   end)
 
   it('preserves newlines', function()
-    if helpers.pending_win32(pending) then return end  -- TODO: Need `cat`.
-    nvim('command', "let j = jobstart(['cat', '-'], g:job_opts)")
-    nvim('command', 'call jobsend(j, "a\\n\\nc\\n\\n\\n\\nb\\n\\n")')
-    eq({'notification', 'stdout',
-      {0, {'a', '', 'c', '', '', '', 'b', '', ''}}}, next_msg())
+    if iswin() then
+      -- find.exe can't handle 3+ consecutive newlines consistently
+      nvim('command', "let j = jobstart(['find', '/v', ''], g:job_opts)")
+      nvim('command', [[call jobsend(j, "a\n\nc\n\n")]])
+      expect_msg_seq(
```

Comment:
If it's not consistent then just mark it pending. The entire purpose of this test is to check that newlines are preserved (consistently).

Reply:
I'll revert it back to pending. Revist when testing with cat.exe gets better.